_sass/base/_reset.scss

```diff
@@ -20,7 +20,11 @@ body {
    	line-height: 1;
 }
 
-ol, ul {
+ol {
```

Comment:
Without this change, ordered lists wouldn't show any numbers when generated from the markdown text. Not sure if there's a better way to fix it